src/demo/index.tsx

```diff
@@ -19,54 +19,61 @@ import { LauncherApp } from './apps/LauncherApp';
  * applications, likely made by different vendors, that are both capable of providing the same funcionality.
  */
 
-let uuid: string, color: string;
-let app: JSX.Element;
 
-if (window.hasOwnProperty("fin")) {
-    uuid = fin.desktop.Application.getCurrent().uuid;
-
-    //Colours are used to simulate having multiple apps capable of handling the same intent.
-    //In this sample they are re-skins of the same application, but in a real scenario they would be completely unrelated applications.
-    color = uuid.split("-")[2];
+function App(): React.ReactElement {
+    const hasFin = window.hasOwnProperty("fin");
+    let uuid = fin.Window.me.uuid;
```

Comment:
would this error if `hasFin` is `undefined`?

Reply:
Removed the check as it is unneeded.